src/main/native/jni_utils.h

```diff
@@ -2,12 +2,40 @@
 #define _Included_org_bblfsh_client_libuast_Libuast_jni_utils
 
 #include <jni.h>
+#include <string>
 
 extern const char *CLS_NODE;
 extern const char *CLS_CTX;
+extern const char *CLS_OBJ;
```

Comment:
Some of this is pre-existing, but consider documenting these names here in the header (this is what will be included by anywhere it's used, so this is presumably what the reader is going to look at).

A the least, please consider leaving a block comment nearby these definitions, to describe what they're used for. Some of the names are kind of obvious, but for example it isn't obvious that these are Babelfish types rather than Java ones, for example.